.storybook/stories/11-KaraokeMode.stories.tsx

```diff
@@ -23,22 +24,18 @@ interface KaraokeModeProps {
 export const KaraokeMode: React.FC<KaraokeModeProps> = args => {
 	const { wrapper } = useFilePlayerStyles().classes;
 
+	const { videoContextApi, setVideoContext } = usePlayerContext();
+
 	const [videoDuration, setVideoDuration] = useDelayedState<number>(0);
-	const [isContextReady, setIsContextReady] = React.useState(false);
 	const [isPlaying, setIsPlaying] = useDelayedState(false);
 	const [transcript, setTranscript] = useDelayedState<Transcript[]>([]);
 
 	const videoContextRef = React.useRef<VideoContext>();
```

Comment:
Do we still need this? Esp. can we get `videoContextRef?.current?.reactPlayerRef?.current?.getInternalPlayer().currentTime` also via the `videoContextApi` instead? (sth call `getInternal` sounds a bit suspicious...)

Reply:
Hmm, I also think that videoContextRef wouldnt be so "popular" in usage, but in case if will be needed work with ReactPlayer instance ref, or maybe getting fullscreen status would be handy in some cases.
but of course, the most popular - videoContextApi